web/src/components/overlay.tsx

```diff
@@ -269,15 +228,118 @@ const Overlay = ({ setStampDatas }: Overlay): JSX.Element => {
     }
   };
 
+  // const [stampId] = useState(Array(6).fill(1));
+  // var flag = 0;
+  // const interval = 50;
+  // useInterval(() => {
+  //   mtime += interval;
+  //   if (mtime / 1000 - time >= 1) {
+  //     setTime(time + 1);
+  //     getStampDatas();
+  //   }
+  //   stampDatas.map(({ stamp_id }) => {
+  //     if (flag == 0) {
+  //       stampId.fill(0);
+  //       flag = 1;
+  //     }
+  //     if (0 < stamp_id && stamp_id < 7) {
+  //       stampId[stamp_id - 1] += 1;
+  //     }
+  //   });
+  // }, interval);
+
+  // const data = {
+  //   labels: ["👏", "😡", "💕", "🎶", "😱", "🥺"],
+  //   datasets: [
+  //     {
+  //       data: [
+  //         stampId[0],
+  //         stampId[1],
+  //         stampId[2],
+  //         stampId[3],
+  //         stampId[4],
+  //         stampId[5],
+  //       ],
+  //       backgroundColor: [
+  //         "#feca57",
+  //         "#ff6b6b",
+  //         "#ff9ff3",
+  //         "#00d2d3",
+  //         "#5f27cd",
+  //         "#54a0ff",
+  //       ],
+  //     },
+  //   ],
+  // };
+
+  const [stampCount] = useState(Array(10));
+  const [stampId] = useState(Array(6).fill(1));
+
+  var flag1 = 0;
+  var flag2 = 0;
   const interval = 50;
   useInterval(() => {
     mtime += interval;
     if (mtime / 1000 - time >= 1) {
       setTime(time + 1);
       getStampDatas();
+      stampDatas.map(({ stamp_id }) => {
```

Comment:
`map` is used to transform each element and put them into a new array.
What you want to use is `forEach`?